src/valueType.ts

```diff
@@ -14,27 +14,31 @@ export const valueTypeStringForConfig = (config: Config) => {
 
 export const valueTypeString = (valueType: ConfigValueType) => {
   switch (valueType) {
-    case 1:
+    case ConfigValueType.Int:
       return 'int'
-    case 2:
+    case ConfigValueType.String:
       return 'string'
-    case 3:
+    case ConfigValueType.Bytes:
       return 'bytes'
-    case 4:
+    case ConfigValueType.Double:
       return 'double'
-    case 5:
+    case ConfigValueType.Bool:
       return 'bool'
-    case 7:
+    case ConfigValueType.LimitDefinition:
       return 'limitDefinition'
-    case 9:
+    case ConfigValueType.LogLevel:
       return 'logLevel'
-    case 10:
+    // @ts-expect-error The sdk-node seems incorrectly typed for it's ConfigValueType.StringList value
+    case 'STRING_LIST':
```

Comment:
This will require some extra investigation in the node library as well. Ideally, this won't be necessary once that is resolved.